sdk/daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/v1/ledgerinteraction/GrpcLedgerClient.scala

```diff
@@ -348,9 +347,20 @@ class GrpcLedgerClient(val grpcClient: LedgerClient, val applicationId: Option[R
       .map(_.party)
   }
 
-  override def listKnownParties()(implicit ec: ExecutionContext, mat: Materializer) = {
-    grpcClient.partyManagementClient
-      .listKnownParties()
+  override def listKnownParties()(implicit
```

Comment:
Does the v1/v2 correspond to the script versions or is it a remnant of the ledger api v1 versions?

Reply:
It corresponds to the script versions.